data-prepper-core/src/test/resources/valid_multiple_pipeline_configuration.yml

```diff
@@ -1,8 +1,7 @@
 # this configuration file is solely for testing formatting
 test-pipeline-1:
   source:
-    file:
-      path: "/tmp/file-source.tmp"
+    stdin:
```

Comment:
What is the error with file source here? Asking b/c we might run into the error again when stdin is being migrated to use Event model

Reply:
This is actually a problem with how the plugin is being loaded and the fact that I changed to using `@DataPrepperPluginConstructor` and PluginSetting is not being used anymore. StdinSink still uses PluginSetting so it works. We should have a separate issue for this bug as these tests should be refactored.

Reply:
Resolved. Validation of path existing was breaking the test